test/unit/sql/delete.test.js

```diff
@@ -45,5 +45,80 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
       });
 
     });
+
+    suite('include the correct schema during bulk delete when appropriate', function () {
+
+      var User = current.define('test_user', {
+        id: {
+          type:       Sequelize.INTEGER,
+          primaryKey: true,
+          field:      "test_user_id",
+        }
+      }, {
+        schema: 'web',
+        freezeTableName: true,
+        timestamps:false
+      });
+
+      var options = {
+        table: 'test_user',
+        schema: 'web',
+        where: {
+          'test_user_id': 100
+        }
+      };
+
+      test(util.inspect(options, {depth: 2}), function () {
+        return expectsql(
+          sql.deleteQuery(
+            options.table,
+            options.where,
+            options,
+            User
+          ), {
+            postgres: 'DELETE FROM "web"."test_user" WHERE "test_user_id" IN (SELECT "test_user_id" FROM "web"."test_user" WHERE "test_user_id" = 100 LIMIT 1)',
+            sqlite:   'DELETE FROM `test_user` WHERE `test_user_id` = 100',
+            mssql:    'DELETE TOP(1) FROM [test_user] WHERE [test_user_id] = 100; SELECT @@ROWCOUNT AS AFFECTEDROWS;',
+            default:  'DELETE FROM [test_user] WHERE [test_user_id] = 100 LIMIT 1'
+          }
+        );
+      });
```

Comment:
Huh, looks like the tests **do** fail... I will do what I can when I can, but my time will be limited for the next few days